### PR TITLE
Simple New Here? dialog

### PR DESCRIPTION
**Pull request by AlexNisnevich**
image

image

It's just a set of links for now. In the future, we could also track the user's progress in completing these.

### CODE REVIEW

File: src/common/containers/App.tsx [hunk 1]
```diff
@@ -170,6 +171,7 @@ class App extends React.Component<AppProps, AppState> {
         <ErrorBoundary>
           <Switch>
             <Route exact path="/" component={Home} />
+            <Route path="//:dialog" component={Home} />
```
Comment (jacobnisnevich) [inline src/common/containers/App.tsx]: Why two slashes?

File: src/common/containers/Home.tsx [hunk 2]
```diff
@@ -118,6 +125,10 @@ class Home extends React.Component<HomeProps> {
       </div>
     </PaperButton>
   )
+
+  private handleClickNewHere = () => {
+    RouterDialog.openDialog(this.props.history, 'new-here');
```
Comment (jacobnisnevich) [inline src/common/containers/Home.tsx]: Why does this need to have a route? Are we going to link people to `/new-here`?

File: src/common/containers/Home.tsx [hunk 3]
```diff
@@ -81,6 +83,11 @@ class Home extends React.Component<HomeProps> {
 
         <RecentCardsCarousel history={history} />
 
+        <div className="new-here-robot" onClick={this.handleClickNewHere}>
+          <div className="speech-bubble" style={{ fontFamily: 'Carter One', fontSize: 20, color: '#f44336', WebkitTextStroke: '0.5px black' }}>New here?</div>
+          <img src="https://i.imgur.com/e8BYAok.png" alt="New here?" style={{ transform: 'rotate(-45deg)' }} />
```
Comment (jacobnisnevich) [inline src/common/containers/Home.tsx]: Should we be hosting this img?

File: styles/index.css [hunk 4]
```diff
@@ -181,6 +181,32 @@ a.topLink:hover {
   transition: visibility 100ms ease-in-out, opacity 100ms ease-in-out;
 }
 
+/* based on https://leaverou.github.io/bubbly/ */
+.speech-bubble {
+	position: relative;
+	background: white;
+  border-radius: .4em;
```
Comment (jacobnisnevich) [inline styles/index.css]: Some inconsistent spacing here

File: src/common/containers/App.tsx (hunk 1 above)
Comment (AlexNisnevich) [reply src/common/containers/App.tsx]: It's because `RouterDialog`s use double-slashes in the URL.

File: src/common/containers/Home.tsx (hunk 2 above)
Comment (AlexNisnevich) [reply src/common/containers/Home.tsx]: I think it would be reasonable to have the option of linking to `//new-here`.

**Pull request merged**